.eslintignore

```diff
@@ -14,3 +14,5 @@ types/.eslintrc.js
 types/generated
 types/**/*.test-d.ts
 scripts/dockolith
+
+src/v3/
```

Comment:
This is because we bring our own linting, right? Don't want to double-report or fail based on G2 lint configs

Reply:
Yeah that's right